examples/linear_model/plot_lasso_model_selection.py

```diff
@@ -1,171 +1,232 @@
+# Lasso via coordinate descent
+# ............................
+# Let's start by making the hyperparameter tuning using
+# :class:`~sklearn.linear_model.LassoCV`.
+from sklearn.linear_model import LassoCV
+
+start_time = time.time()
+model = make_pipeline(StandardScaler(), LassoCV(cv=20)).fit(X, y)
+fit_time = time.time() - start_time
+
+# %%
+import matplotlib.pyplot as plt
 
-# Display results
-plt.figure()
 ymin, ymax = 2300, 3800
-plt.semilogx(model.alphas_ + EPSILON, model.mse_path_, ":")
+plt.semilogx(model[-1].alphas_, model[-1].mse_path_, linestyle=":")
 plt.plot(
-    model.alphas_ + EPSILON,
-    model.mse_path_.mean(axis=-1),
-    "k",
+    model[-1].alphas_,
+    model[-1].mse_path_.mean(axis=-1),
+    color="black",
     label="Average across the folds",
     linewidth=2,
 )
-plt.axvline(
-    model.alpha_ + EPSILON, linestyle="--", color="k", label="alpha: CV estimate"
-)
-
-plt.legend()
+plt.axvline(model[-1].alpha_, linestyle="--", color="black", label="alpha: CV estimate")
 
+plt.ylim(ymin, ymax)
 plt.xlabel(r"$\alpha$")
 plt.ylabel("Mean square error")
-plt.title(
-    "Mean square error on each fold: coordinate descent (train time: %.2fs)"
-    % t_lasso_cv
+plt.legend()
+_ = plt.title(
+    f"Mean square error on each fold: coordinate descent (train time: {fit_time:.2f}s)"
 )
-plt.axis("tight")
-plt.ylim(ymin, ymax)
 
-# #############################################################################
-# LassoLarsCV: least angle regression
+# %%
+# Lasso via least angle regression
+# ................................
+# Let's start by making the hyperparameter tuning using
+# :class:`~sklearn.linear_model.LassoLarsCV`.
+from sklearn.linear_model import LassoLarsCV
 
-# Compute paths
-print("Computing regularization path using the Lars lasso...")
-t1 = time.time()
-model = LassoLarsCV(cv=20, normalize=False).fit(X, y)
-t_lasso_lars_cv = time.time() - t1
+start_time = time.time()
+model = make_pipeline(StandardScaler(), LassoLarsCV(cv=20, normalize=False)).fit(X, y)
+fit_time = time.time() - start_time
 
-# Display results
-plt.figure()
-plt.semilogx(model.cv_alphas_ + EPSILON, model.mse_path_, ":")
+# %%
+plt.semilogx(model[-1].cv_alphas_, model[-1].mse_path_, ":")
 plt.semilogx(
-    model.cv_alphas_ + EPSILON,
-    model.mse_path_.mean(axis=-1),
-    "k",
+    model[-1].cv_alphas_,
+    model[-1].mse_path_.mean(axis=-1),
+    color="black",
     label="Average across the folds",
     linewidth=2,
 )
-plt.axvline(model.alpha_, linestyle="--", color="k", label="alpha CV")
-plt.legend()
+plt.axvline(model[-1].alpha_, linestyle="--", color="black", label="alpha CV")
 
+plt.ylim(ymin, ymax)
 plt.xlabel(r"$\alpha$")
 plt.ylabel("Mean square error")
-plt.title("Mean square error on each fold: Lars (train time: %.2fs)" % t_lasso_lars_cv)
-plt.axis("tight")
-plt.ylim(ymin, ymax)
-
-plt.show()
+plt.legend()
+_ = plt.title(f"Mean square error on each fold: Lars (train time: {fit_time:.2f}s)")
+
+# %%
+# Summary of cross-validation approach
```

Comment:
A final summary with comparison of CV and AIK/BIC results is missing. At least one or two sentences.